jkube-kit/generator/webapp/src/main/java/org/eclipse/jkube/generator/webapp/WebAppGenerator.java

```diff
@@ -180,7 +180,7 @@ private AppServerHandler createCustomAppServerHandler(String from) {
 
   private AssemblyConfiguration createAssembly(AppServerHandler handler) {
     final File sourceFile = Objects.requireNonNull(JKubeProjectUtil.getFinalOutputArtifact(getProject()),
-        "Final output artifact file was not detected");
+        "Final output artifact file was not detected. The project may have not been built. HINT: try to execute the `build` task prior to `ocBuild` task.");
```

Comment:
This message is shown both for Maven and Gradle plugins.

Something more generic would be more suitable:
```suggestion
        "Final output artifact file was not detected. The project may have not been built. HINT: try to compile and package your application prior to running the container image build task.");        
```

^^^ @sunix

Reply:
Alright, do you want me to proceed with the change or should I wait for @sunix 's review message?

Reply:
Alright, will do